types/got/index.d.ts

```diff
@@ -100,7 +100,8 @@ declare namespace got {
     }
 
     interface GotJSONOptions extends GotOptions<string | null> {
-        body?: any;
+        // Body must be an object. See https://github.com/sindresorhus/got/issues/511
+        body?: object;
```

Comment:
This is not correct. Read the docs: https://github.com/sindresorhus/got#json

> body must be a plain object **or array** and will be stringified.

Reply:
Array is assignable to `object` type: http://www.typescriptlang.org/play/index.html#src=let%20arr%3A%20object%20%3D%20%5B%5D%3B